cdap-ui/app/cdap/components/PipelineTriggers/index.js

```diff
@@ -82,8 +83,19 @@ export default class PipelineTriggers extends Component {
       },
     });
 
+    PipelineTriggersStore.dispatch({
+      type: PipelineTriggersActions.setWorkflowName,
+      payload: {
+        workflowName: GLOBALS.programId[this.props.pipelineType],
+      },
+    });
```

Comment:
Can you see if the action `setPipelineName` being called anywhere else? It would be better to merge the payloads of setPipelineName and setWorkflowName into a single action.

Reply:
Fixed by consolidating down to one dispatch call